### PR TITLE
[EN/KO] Update Medals

### PR DESCRIPTION
**Pull request by dora-0**

### CODE REVIEW

**Comment by dora-0** [comment]
Ready for reviews

File: wiki/Medals/en.md [hunk 1]
```diff
@@ -33,15 +33,15 @@ The **Skill** medals are currently divided into three groups: Combo, High Ranker
 
 The Combo medals can be obtained by gaining a specific [combo](/wiki/combo) in osu!standard and can be completed with any one song. Achieving the same combo count in [osu!taiko](/wiki/osu!taiko), [osu!catch](/wiki/osu!catch), or [osu!mania](/wiki/osu!mania) mode will not result in getting the medal.
 
-For the 1,000 and 2,000 combo medals, you should find approved beatmaps instead of ranked beatmaps. However, **do not search for or play unapproved beatmaps!**
+For the 1,000 and 2,000 combo medals, you should find long beatmaps. **If the beatmap is not in ranked or approved state, it's not recognized.**
```
Comment (TPGPL) [inline wiki/Medals/en.md]:
```suggestion
For the 1,000 and 2,000 combo medals, you should look for sufficiently long beatmaps. **If the beatmap is not in ranked or approved state, the medals will not be granted.**
```

File: wiki/Medals/en.md [hunk 2]
```diff
@@ -33,15 +33,15 @@ The **Skill** medals are currently divided into three groups: Combo, High Ranker
...
-The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing any map of a certain star difficulty. The *S* medals will be obtained if you pass the map with a S ranking whilst also having an [FC](/wiki/FC). The difficulty of the map will round to the nearest star difficulty and will give the medal for what it rounds to. You can pass or get a S on a map with difficulty increasing mods or difficulty decreasing mods. The map must also be ranked to be able to earn a medal.
+The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing any map of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be obtained if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the map will round to the nearest star difficulty and will give the medal for what it rounds to. For example, if you haven't earned 6/7 star medals but clear 7.8 stars, you'll only get 7 star medals. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The map must also be ranked to be able to earn a medal.
```
Comment (TPGPL) [inline wiki/Medals/en.md]:
```suggestion
The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing a beatmap of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be granted if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the map will be rounded to the nearest star difficulty and will grant the medal for what it is rounded to. For example, if you haven't earned 6/7 star medals but cleared 7.8 stars difficulty, you will only receive a 7 star medal. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The beatmap must also be ranked to grant a medal.
```

File: wiki/Medals/en.md [hunk 3]
```diff
@@ -33,15 +33,15 @@ The **Skill** medals are currently divided into three groups: Combo, High Ranker
...
+The Star-based achivements are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing a beatmap of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be granted if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the map will be rounded to the nearest star difficulty and will grant the medal for what it is rounded to. For example, if you haven't earned 6/7 star medals but cleared 7.8 stars difficulty, you will only receive a 7 star medal. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The beatmap must also be ranked to grant a medal.
```
Comment (MegaApplePi) [inline wiki/Medals/en.md]:
```suggestion
The star-based medals are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing a beatmap of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be granted if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the beatmap will be rounded to the nearest star difficulty and will grant the medal for what it is rounded to. For example, if you have not earned the 6/7 star medals but cleared 7.8 stars difficulty, you will only receive a 7 star medal. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The beatmap must also be ranked to grant a medal.
```

- Lowercase star-based
- achivements (typo) -> medals
- map -> beatmap
- haven't -> have not

Comment (MegaApplePi) [reply wiki/Medals/en.md]: I'm a bit confused with the example about not have had earned the 6/7 star medals(?) but completing a 7.8 star medal map will give you the 7 star medal?

Comment (dora-0) [reply wiki/Medals/en.md]: I think it doesn't earn 6, 7 stars medal at the same time. But I've never done that before, so I'm not sure :thinking:

Comment (cl8n) [reply wiki/Medals/en.md]: the point there is that getting the 7 star medal doesn't also give you the 6 star one--- this should probably be explained in a way that's not just a random example

File: wiki/Medals/en.md [hunk 4]
```diff
@@ -33,15 +33,15 @@ The **Skill** medals are currently divided into three groups: Combo, High Ranker
 
 The Combo medals can be obtained by gaining a specific [combo](/wiki/combo) in osu!standard and can be completed with any one song. Achieving the same combo count in [osu!taiko](/wiki/osu!taiko), [osu!catch](/wiki/osu!catch), or [osu!mania](/wiki/osu!mania) mode will not result in getting the medal.
 
-For the 1,000 and 2,000 combo medals, you should find approved beatmaps instead of ranked beatmaps. However, **do not search for or play unapproved beatmaps!**
+For the 1,000 and 2,000 combo medals, you should look for sufficiently long beatmaps. **If the beatmap is not in ranked or approved state, the medals will not be granted.**
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Medals/en.md]: also capitalize Ranked/Approved

[the remark this answers is not in the document]
Comment (MegaApplePi) [reply wiki/Medals/en.md]: the suggestion was updated

File: wiki/Medals/en.md (hunk 3 above)
Comment (dora-0) [reply wiki/Medals/en.md]: I think we can just delete the example section

**Pull request merged**